lib/galaxy/util/bool_expressions.py

```diff
@@ -136,7 +135,7 @@ def __init__(self, evaluator: TokenEvaluator, token_format: Optional[str] = None
         action.evaluator = evaluator
         boolOperand = TRUE | FALSE | Word(token_format or DEFAULT_TOKEN_FORMAT)
         boolOperand.setParseAction(action)
-        self.boolExpr: Forward = infixNotation(
+        self.boolExpr = infixNotation(
```

Comment:
I fixed it locally with:

```suggestion
        self.boolExpr: ParserElement = infixNotation(
```

Not sure if it matters.

Reply:
I guess it doesn't matter since `mypy` infers it to *ParserElement* anyway but it also doesn't hurt to be more explicit :)